api/hasura/event_triggers/eventManager.ts

```diff
@@ -35,6 +30,12 @@ const STAGING_HANDLERS: HandlerDict = {
   refundGiveTelegram,
```

Comment:
I'm not 100% sure on the dependencies here but do we need deleteUser to be impld ?

Reply:
theres an issue because if admin updates user to opt out the refund wouldn't happen